src/index.html

```diff
@@ -1,17 +1,206 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
+    <meta 
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="formElement" for="surname">
```

Comment:
Problem with styling a label like this is that we get a label that is wider than the text describing an input. You can see that if you click on the right side of the form, one of the inputs will be selected.

A better approach would be to wrap label and input in a div and style that div.